bsoid_app/predict.py

```diff
@@ -47,173 +44,130 @@ def __init__(self, ROOT_PATH, DATA_DIRECTORIES, input_filenames, processed_input
-            if st.checkbox('New root directory (not {}) for these new files?'.format(self.root_path), False, key='pn'):
-                self.new_root_path = st.text_input('Enter a __root directory__, e.g. __/Users/projectX__', os.getcwd())
-            else:
-                self.new_root_path = self.root_path
-            num_dir = int(st.number_input('How many __data containing directories__ '
-                                          'under {} for B-SOiD predictions?'.format(self.new_root_path), value=3))
-            st.markdown('Your will predict on data files in *{}* directories.'.format(num_dir))
-            self.filetype = st.selectbox('What type of file are these new data?', ('csv', 'h5', 'json'),
-                                         index=int([i for i, s in enumerate(['csv', 'h5', 'json'])
-                                                    if s in self.input_filenames[0].partition('.')[-1]][0]))
-            for i in range(num_dir):
-                new_directory = st.text_input('Enter # {} __data file containing directory__ under {}, '
-                                              'e.g. __/control__ for /Users/projectX/control/xxx.{}'
-                                              ''.format(i + 1, self.new_root_path, self.filetype))
-                try:
-                    os.listdir(str.join('', (self.new_root_path, new_directory)))
-                except FileNotFoundError:
-                    st.error('No such directory')
-                if not new_directory in self.new_directories:
-                    self.new_directories.append(new_directory)
-            st.markdown('You have selected **{}** as your _sub-directory(ies)_.'.format(self.new_directories))
-            st.write('Average video frame-rate for xxx.{} pose estimate files.'.format(self.filetype))
-            self.new_framerate = int(st.number_input('What is your frame-rate?', value=self.framerate))
-            st.markdown('You have selected **{} frames per second**.'.format(self.new_framerate))
-        if st.checkbox('For every new dataset, you would want to change the PREFIX so it does not overwrite previous '
-                       'predictions. Would you like to change the PREFIX? Currently it is set to save'
-                       ' as **{}/{}_predictions.sav**.'.format(self.working_dir, self.prefix), False, key='pp'):
-            today = date.today()
-            d4 = today.strftime("%b-%d-%Y")
-            self.new_prefix = st.text_input('Enter new prediction variable PREFIX:', d4)
-            if self.new_prefix:
-                st.markdown('You have chosen **{}_predictions.sav** for new predictions.'.format(self.new_prefix))
-            else:
-                st.error('Please enter a name for your new prediction variable PREFIX.')
-        else:
-            self.new_prefix = self.prefix
+        self.new_root_path = self.root_path
+        self.filetype = [s for i, s in enumerate(['csv', 'h5', 'json'])
```

Comment:
Yeah line 34 won't work

Reply:
removed these lines, we don't need them. self.filetype is defined in setup